nf_core/modules/modules_json.py

```diff
@@ -1249,15 +1249,15 @@ def recreate_dependencies(self, repo, org, subworkflow):
         i.e., no module or subworkflow has been installed by the user in the meantime
         """
 
-        sw_name = subworkflow["name"] if isinstance(subworkflow, dict) else subworkflow
+        sw_name = subworkflow["name"]
         sw_path = Path(self.subworkflows_dir, org, sw_name)
         dep_mods, dep_subwfs = get_components_to_install(sw_path)
         assert self.modules_json is not None  # mypy
         for dep_mod in dep_mods:
             name = dep_mod["name"]
-            if dep_mod["git_remote"] is not None:
-                current_repo = dep_mod["git_remote"]
-                current_org = dep_mod["org_path"]
+            if dep_mod.get("git_remote") is not None:
+                current_repo = dep_mod.get("git_remote", repo)
+                current_org = dep_mod.get("org_path", org)
```

Comment:
Like for the branch, f there is "git_remote", then there must be "org_path" too ?
```suggestion
                current_org = dep_mod["org_path"]
```